XmlFormatter/src/Logging/Strategies/SimpleFileLogStrategy.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.IO;
+using XmlFormatter.src.Interfaces.Logging;
+
+namespace XmlFormatter.src.Logging.Strategies
+{
+    /// <summary>
+    /// This simple logger will write the data into a given file
+    /// </summary>
+    class SimpleFileLogStrategy : ILoggingStrategy
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Implement 'IDisposable' in this class and use the 'Dispose' method to call 'Dispose' on 'streamWriter'.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5095168)